jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/test/java/org/jbpm/workbench/es/client/editors/errorlist/ExecutionErrorListPresenterTest.java

```diff
@@ -259,4 +277,71 @@ public void testActiveSearchFiltersProcessInstanceId(){
 
     }
 
+    @Test
+    public void testGoToJobErrorSummary() {
+        String jobId = "1";
+
+        presenter.goToJob(ExecutionErrorSummary.builder().jobId(Long.valueOf(jobId)).build());
+
+        final ArgumentCaptor<PlaceRequest> captor = ArgumentCaptor.forClass(PlaceRequest.class);
+        verify(placeManager).goTo(captor.capture());
+        final PlaceRequest request = captor.getValue();
+        assertEquals(JOBS, request.getIdentifier());
+        assertEquals(jobId, request.getParameter(PerspectiveIds.SEARCH_PARAMETER_JOB_ID, null));
+    }
+
+    @Test
+    public void testGoProcessInstanceErrorSummary() {
+
+        ExecutionErrorSummary errorSummary =
+                ExecutionErrorSummary.builder()
+                        .deploymentId("test_depId")
+                        .processInstanceId(Long.valueOf("1"))
+                        .processId("test_processId")
+                        .build();
+
+        presenter.goToProcessInstance(errorSummary);
+        final ArgumentCaptor<PlaceRequest> captor = ArgumentCaptor.forClass(PlaceRequest.class);
+        verify(placeManager).goTo(captor.capture());
+        final PlaceRequest request = captor.getValue();
+        assertEquals(PROCESS_INSTANCES, request.getIdentifier());
+        assertEquals(errorSummary.getProcessInstanceId().toString(), request.getParameter(PerspectiveIds.SEARCH_PARAMETER_PROCESS_INSTANCE_ID, null));
+    }
+
+    @Test
+    public void testViewJobsActionCondition() {
+        doAnswer(new PerspectiveAnswer(JOBS)).when(authorizationManager).authorize(any(ResourceRef.class), eq(identity));
+
+        assertTrue(presenter.getViewJobActionCondition().test(ExecutionErrorSummary.builder().jobId(Long.valueOf(1)).build()));
+
+        when(authorizationManager.authorize(any(ResourceRef.class), eq(identity))).thenReturn(false);
+
+        assertFalse(presenter.getViewJobActionCondition().test(new ExecutionErrorSummary()));
+    }
```

Comment:
@bsremac I think that the condition is there based on if the jobid isn't null. For instances an error on user task won't have it set.